app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -26,21 +35,51 @@ export const ListingDetailsAdditionalInformation = ({
             <TextTruncate text={listing.Listing_Other_Notes} />
           </div>
         )}
-        <div className="info-card bg-gray-100 border-0">
-          <h3 className="text-serif-lg">{t("listings.requiredDocuments")}</h3>
-          <div className="text-sm">
-            <TextTruncate text={listing.Required_Documents} />
+        {listing.Required_Documents && (
```

Comment:
Looks like this section is missing a [paragraph](https://github.com/SFDigitalServices/sf-dahlia-web/blob/83ce5a75469b737a8c415a5d92d862b68b6cfe3e/app/assets/javascripts/listings/components/listing/legal-section.html.slim#L20-L20) for `listings.requiredDocumentsAfterApplying`